src/library/librarycontrol.cpp

```diff
@@ -166,6 +166,30 @@ LibraryControl::LibraryControl(Library* pLibrary)
                 &LibraryControl::slotMoveFocus);
     }
 
+    // Controls to move tracks on playlist
+    // Track move controls (emulate Alt+up/down button press)
+    m_pMoveTrackUp = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "MoveTrack-Up"));
+    m_pMoveTrackDown = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "MoveTrack-Down"));
```

Comment:
Let's rename so the controls match the scheme of `MoveUp`, `ScrollUp` etc.
```suggestion
    m_pMoveTrackUp = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "MoveTrackUp"));
    m_pMoveTrackDown = std::make_unique<ControlPushButton>(
            ConfigKey("[Library]", "MoveTrackDown"));
```